swipe-keyboard.js

```diff
@@ -6,6 +6,8 @@ const swipeKeyboard = {
   touchPath: [],
   swipes: [],
   moves: [],
+  uMoveMode: undefined, // 'top' | 'bottom' Unlike lrMoveMode, this is only used in "neutral R" position
+  lrMoveMode: undefined, // 'left' | 'right'
```

Comment:
(Comment is about commit da485e13c047d7b72e9572094cd9c7792a3586db `Fix the "started on quadrant edge" problem` generally)

Looks like this works great! I think the main remaining bits of unintuitiveness are:

- R U' R' type inserts (as opposed to R U R' type) are reverse of intuitive -- but easy enough to adapt to
- In some cases, (e.g. red blue first pair on scramble 20) it's not immediately obvious whether you want to start swiping on the top or bottom. Then, mid gesture, you do a neutral-r U move and you have to think about which direction it'll go